recipes-example/template-dbus-service/template-dbus-service_git.bb

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2019 Luxoft Sweden AB
+# SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCPV}"
```

Comment:
this is unnecessary as it's not `1.0` and `${PV}` is now set to `git+bla`

Reply:
I don't agree, but the version was incorrect.

Reply:
You either set in in the recipe or its name; doubling information makes no sense: at some point, someone will forget to modify it you'll end up with a package that has a wrong version.